assets/js/lib/test-utils/factories/advisoryErrata.js

```diff
@@ -0,0 +1,25 @@
+import { faker } from '@faker-js/faker';
+import { Factory } from 'fishery';
+import { advisoryType } from './relevantPatches';
+
+export const advisoryErrataFactory = Factory.define(() => ({
+  id: faker.number.int({ min: 1, max: 65536 }),
+  issue_date: faker.date.recent({ days: 30 }),
+  update_date: faker.date.recent({ days: 30 }),
+  last_modified_date: faker.date.recent({ days: 30 }),
+  synopsis: faker.lorem.sentence(),
+  release: faker.number.int({ min: 1, max: 256 }),
+  advisory_status: 'stable',
+  vendor_advisory: faker.lorem.word(),
+  type: faker.helpers.arrayElement(advisoryType),
+  product: faker.internet.userName(),
+  // FIXME: why is this cased in CamelCalse?
+  erratFrom: faker.lorem.word(),
```

Comment:
To answer the `FIXME`; this is what is returned from SUMA's [getDetails](https://documentation.suse.com/suma/4.3/api/suse-manager/api/errata.html#apidoc-errata-getDetails-loggedInUser-advisoryName) endpoint.

However we translate this field into snake case when it is returned from Trento. So it should be like:

```suggestion
  errata_from: faker.lorem.word(),
```